cl/favorites/utils.py

```diff
@@ -167,7 +166,7 @@ async def get_top_prayers() -> QuerySet[RECAPDocument]:
                 When(prayeravailability__id__isnull=False, then=Value(True)),
                 default=Value(False),
             ),
-            last_checked=F("prayeravailability__last_checked"),
+            last_checked=TruncDate("prayeravailability__last_checked"),
```

Comment:
Instead of the `TruncDate` class, we can achieve the same result more directly by accessing the date part of the `prayeravailability__last_checked` field using:

```suggestion
            last_checked=F("prayeravailability__last_checked__date"),
```